app/assets/src/components/views/SampleViewV2/report_table.scss

```diff
@@ -92,11 +101,13 @@
 }
 
 .genusRow,
-.speciesRow {
-  border-left: 5px solid transparent;
+.speciesRow,
+.row {
+  // border-left: 5px solid transparent;
   &.highlighted {
     background-color: $primary-lightest;
-    border-left: 5px solid $primary-lighter;
+    // border-left: 5px solid $primary-lighter;
+    box-shadow: inset 5px 0 $primary-lighter;
```

Comment:
Was `border-left` not working? `box-shadow` seems a little unconventional. Also remove commented lines.

Reply:
See above.